ion/index.html

```diff
@@ -27,6 +27,7 @@
 </head>
 
   <body>
+    <h1> Hola </h1>
```

Comment:
Is this needed?

Reply:
No, sorry, it must have slipped. I will delete it